packages/extension/src/components/icon/keystone.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from "react";
+import { IconProps } from "./types";
+
+export const KeystoneIcon: FunctionComponent<IconProps> = ({
```

Comment:
@renfengshi Could you look at another icon file and follow the convention for icon files?